cloud_functions/ValidationError.js

```diff
@@ -0,0 +1,9 @@
+class ValidationError extends Error {
```

Comment:
What's the rationale for creating this subclass? It doesn't look like we have any logic that makes decisions based on it.

Reply:
Ah. I had done something with this at one point, but it looks like it turns out to not be needed. I'll probably remove it (after double-checking that I didn't miss an intended usage).